native/cocos/platform/ios/IOSPlatform.h

```diff
@@ -44,6 +44,10 @@ class IOSPlatform : public UniversalPlatform {
      * @brief Start base platform initialization.
      */
     int32_t run(int argc, const char **argv) override;
+    
+    void requestQuit();
```

Comment:
Quit or Exit ?
The aforementioned TS code call it `exit`.

Reply:
The process is like this: close（ts）-》quit->onClose（ts）->exit
This interface is for the user to perform a manual exit。
Would it be more confusing to use exit?

Reply:
I meant how about renaming `requestQuit` to `requestExit`, not just name it `exit`.